[{"commit":{"message":"8265261: Fix the section which tests cancellation"},"files":[{"filename":"test\/jdk\/java\/nio\/file\/Files\/InterruptCopy.java"}],"sha":"85f78867544da7f3708c1aa2d172530203a8cd8c"},{"commit":{"message":"8265261: java\/nio\/file\/Files\/InterruptCopy.java fails with java.lang.RuntimeException: Copy was not interrupted"},"files":[{"filename":"test\/jdk\/java\/nio\/file\/Files\/InterruptCopy.java"}],"sha":"7c75978b4b90fae880302d3763a83f7f61138b4c"}]